cg.js

```diff
@@ -0,0 +1,24 @@
+const replace = require('replace-in-file');
```

Comment:
do we want to check this in?

Reply:
Did not mean to include that, its a temporary script I use to automate some of changelog updates. Removed in this commit: https://github.com/cerner/terra-core/pull/1497/commits/84f713d15c2efb6f775dbcde6ee469825a35aafb